index.html

```diff
@@ -275,8 +275,25 @@ <h3 class="im-qaisar">Glad to see you!</h3>
             <li> <img src="./public/adobe-illustrator.svg" alt=""> <span>Adobe Illustrator</span> </li>
           </ul>
         </div>
+
     </section>
+    <section class="form">
+      <h1>Contact Me</h1>
+      <p>If you have an application, you are interested in developing, a feature that you need built or a project that needs coding. I'd love to help with it</p>
+
```

Comment:
Please Remove unwanted space from your code base in order to save memory.

Reply:
Okay noted